src/sutta_publisher/shared/config.py

```diff
@@ -5,19 +5,107 @@
 from typing import cast
 
 import inject
+import requests
 
 from sutta_publisher.publishers.base import ActivePublishers
 from sutta_publisher.publishers.html import HtmlPublisher
 
 
+@dataclass(frozen=True)
+class EditionsConfig:
+    publication_number: str
+    text_uid: str
+    edition_number: str
+    isbn: str
+    cip: str
+    publication_type: str
+    format: str
+    creator: str
+    output: str
+    page_size: str
+    cover_image: str
+    main_toc_depth: str
+    secondary_toc: bool
+    number_of_volumes: int
+    volumes: list
+
+
 @dataclass(frozen=True)
 class Config:
-    bilara_data_url = "https://github.com/suttacentral/bilara-data"
+    PUBLICATION_JSON_RAW_URL = (
+        "https://raw.githubusercontent.com/suttacentral/bilara-data/published/_publication-v2.json"
+    )
+
     publication_number: str
+    source_url: str
+    root_lang_iso: str
+    root_lang_name: str
+    root_title: str
+    creation_process: str
+    text_description: str
+    is_published: str
+    publication_status: str
+
+    creator_uid: str
+    creator_name: str
+    creator_github_handle: str
+
+    text_uid: str
+    translation_lang_iso: str
+    translation_lang_name: str
+    translation_title: str
+    translation_subtitle: str
+
+    license_type: str
+    license_abbreviation: str
+    license_url: str
+    license_statement: str
+    first_published: str
+
+    editions_url: str
+    editions: EditionsConfig
```

Comment:
not sure if we need all of it. I'd leave only 
```
creator_name
translation_title
translation_subtitle
root_title
```
also `editions` but that should be `editions: List[EditionsConfig]`

Reply:
Ok, I removed most of this fields. Apart from those mentioned by you, I left `publication_number` so publication could be easily identified. `source_url` and 'editions_url` also can become handy when we would like to download contents of publication or editions.

Reply:
`editions_url` - it's needed for the config internally yes, but when it's loaded we won't be needing that in application anymore.
with `source_url` also I doubt we will need it - we are going to get all our data (:crossed_fingers: ) from bilara endpoint so also seems like a dead property